packages/pages/src/dev/server/middleware/serverRenderSlugRoute.ts

```diff
@@ -69,7 +69,7 @@ export const serverRenderSlugRoute =
         locale,
         document,
       });
-      sendAppHTML(res, templateModuleInternal, props, vite, url.pathname);
+      sendAppHTML(res, templateModuleInternal, props, vite, `/${slug}`);
```

Comment:
Does the `sendStaticPage` call (line 35) also need to be updated to pass the decoded slug instead of `url.pathname`?

Reply:
I don't think so. I was able to get a static page (with German characters in the path) to load just fine without updating the usage on line 35.